traffic_ops/traffic_ops_golang/cdni/shared.go

```diff
@@ -518,7 +537,14 @@ func checkBearerToken(bearerToken string, inf *api.APIInfo) (string, error) {
 		return "", errors.New("invalid token - incorrect dcdn")
 	}
 	if ucdn == "" {
-		return "", errors.New("invalid token - empty ucdn field")
+		if inf.User.PrivLevel == auth.PrivLevelAdmin {
```

Comment:
PrivLevel is deprecated and not even visible in APIv4 - does this correspond to some Permission?